.github/workflows/ci.yml

```diff
@@ -207,7 +206,7 @@ jobs:
         github.event_name == 'pull_request' &&
         contains(github.event.pull_request.labels.*.name, 'Run publish')
       )
-    needs: [test, docs]
+    needs: [test, docs, publish]
```

Comment:
Since these are failing to upload sometime, I want to make sure the normal wheels get uploaded before the aarch64 wheels are attempted.

Reply:
Also, not sure if I need to report this upstream but both upload jobs are uploading all of the artifacts, regardless of which job created them.

I assume this is not intended behaviour.